keadm/app/cmd/util/ubuntuinstaller.go

```diff
@@ -218,6 +219,118 @@ func (u *UbuntuOS) InstallDocker() error {
 	return nil
 }
 
+//InstallContainerd will install the containerd in the host
+func (u *UbuntuOS) InstallContainerd() error {
+
+	//Add overlay and netfilter modules
+	instPreReq := fmt.Sprintf("modprobe overlay && modprobe br_netfilter")
+	cmd := &Command{Cmd: exec.Command("sh", "-c", instPreReq)}
+	err := cmd.ExecuteCmdShowOutput()
+	errout := cmd.GetStdErr()
+	if err != nil || errout != "" {
+		return fmt.Errorf("%s", errout)
+	}
+	fmt.Println(cmd.GetStdOutput())
+	criconf := fmt.Sprint(`cat > /etc/sysctl.d/99-kubernetes-cri.conf <<EOF
+net.bridge.bridge-nf-call-iptables  = 1
+net.ipv4.ip_forward                 = 1
+net.bridge.bridge-nf-call-ip6tables = 1
+EOF`)
+	cmd = &Command{Cmd: exec.Command("sh", "-c", criconf)}
+	cmd.ExecuteCommand()
+	stdout := cmd.GetStdOutput()
+	errout = cmd.GetStdErr()
+	if errout != "" {
+		return fmt.Errorf("%s", errout)
+	}
+	fmt.Println("Expected certificates and software properties will get installed %s", stdout)
+
+	// Set up the repository
+	// Install packages to allow apt to use a repository over HTTPS
+	instpkg := fmt.Sprint("apt-get update && apt-get install -y apt-transport-https ca-certificates curl software-properties-common")
+	cmd = &Command{Cmd: exec.Command("sh", "-c", instpkg)}
+	cmd.ExecuteCommand()
+	stdout = cmd.GetStdOutput()
+	errout = cmd.GetStdErr()
+	if errout != "" {
+		return fmt.Errorf("%s", errout)
+	}
+	fmt.Println("install package is %s", stdout)
```

Comment:
its better we use standard logger instead of fmt to print all installer logs. can we create an issue to handle the same throughout the installer code ?